application/controllers/handler.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
       registered: (new Date()).toISOString(),
       providers: []
     };
-    console.log('Registration: ', user);
+    console.log('Registration: ', user.username, user.email);
```

Comment:
I will remove this log and log the user with the deleted password if an error occurs. good idea!